.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "VATSIM UK Nexus",
+  "dockerComposeFile": "docker-compose.yml",
+  "service": "nexus",
+  "workspaceFolder": "/app/nexus",
+
+  // Set *default* container specific settings.json values on container create.
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash"
+  },
+
+  // Add the IDs of extensions you want installed when the container is created.
+  "extensions": [
+    "felixfbecker.php-debug",
+    "felixfbecker.php-intellisense",
+    "mrmlnc.vscode-apache",
+    "bradlc.vscode-tailwindcss",
+    "bmewburn.vscode-intelephense-client",
+    "esbenp.prettier-vscode"
+  ],
+
+  // For use with PHP or Apache (e.g.php -S localhost:8080 or apache2ctl start)
+  "forwardPorts": [8080, 3306],
+
+  "postCreateCommand": "composer install --no-interaction && yarn"
```

Comment:
Suppose you could run a migrate or key generate for this? maybe just leave that up to the user

Reply:
I think the one-time stuff is best left up to user for now, but there is lots of scope for automating the setup here